plugins/jobs/src/js/components/JobsBreadcrumbs.js

```diff
@@ -4,62 +4,45 @@ import PropTypes from "prop-types";
 import React from "react";
 import { Tooltip } from "reactjs-components";
 
-import Job from "#SRC/js/structs/Job";
-import JobTree from "#SRC/js/structs/JobTree";
 import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
 import Breadcrumb from "#SRC/js/components/Breadcrumb";
 import BreadcrumbSupplementalContent from "#SRC/js/components/BreadcrumbSupplementalContent";
 import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
 import Icon from "#SRC/js/components/Icon";
 import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
 
-function getLongestRunningTask(item) {
-  if (!(item instanceof Job)) {
+function ItemStatus({ item: { jobRuns } }) {
```

Comment:
🎨 
I think this component knows a lot about the structure of the job. That is why it needs to call `longestRunningActiveRun.tasks.longestRunningTask`.

That is a non blocking for me, but I think it would be cool if it just received the status, and all this how to get status logic would not be part of it.

Reply:
I understand that being out of the scope of this refactor as well